modules/virt-cnv-ssp-alerts.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-events.html/virt-virtualization-alerts.adoc
+
+[id="virt-cnv-ssp-alerts_{context}"]
+= SSP alerts
+
+SSP alerts provide information about problems for the CNV SSP Operator.
+
+//SSPFailingToReconcile Alert
+[id="SSPFailingToReconcile_{context}"]
+== SSPFailingToReconcile alert
+
+.Description
+
+The SSP Operator’s pod is up, but the pod's reconcile cycle consistently fails. This failure includes failure to update the resources it is responsible for, failure to deploy the template validator, or failure to deploy or update the common templates.
+
+.Reason
+
+If the SSP Operator fails to reconcile, then the deployment of dependent components fails, reconciliation of component changes fails, or both. Additionally, the updates to the common templates and template validator reset and fail.
+
+.Troubleshoot
+
+* Check the ssp-operator pod's logs for errors:
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get deployment -A | grep ssp-operator | awk '{print $1}')"
```

Comment:
Same comment about the namespace here. Should this be `openshift-cnv`?

Reply:
Same reply as previous. We initially did change it, but decided to keep it generic for future cases.